config/cable.yml

```diff
@@ -0,0 +1,10 @@
+development:
+  adapter: async
+
+test:
+  adapter: test
+
+production:
+  adapter: redis
```

Comment:
was this auto generated?,  Are we going to use this for real time?

Reply:
Yes , it was generated by rails